### ISSUE TITLE
How to use torch_momentum?

### ISSUE THREAD

**Issue by qingqing01** [PaddlePaddle/Paddle#1002]
There is a `torch_momentum` option in the SgdOptimizer.  If use this option, how to set the learning rate compared with `momentum`  method？

**Comment by lzhao4ever** [comment]
If the learning rate is lr_momentum when use momentum method in the SgdOptimizer, then set the learning rate to be 10 * lr_momentum when use torch_momentum.

**Comment by qingqing01** [comment]
```
    real torch_learningRate = optConfig_.learning_method() == "torch_momentum"
                                  ? 1.0 - paraConfig.momentum()
                                  : 1.0;
    vecs[PARAMETER_VALUE]->sgdUpdate(
        *vecs[PARAMETER_GRADIENT],
        *vecs[PARAMETER_MOMENTUM],
        learningRate_ * paraConfig.learning_rate() *
            (firstTime_ ? 1.0 : torch_learningRate),
        paraConfig.momentum(),
        applyDecay_ ? paraConfig.decay_rate() : 0);
  }
```

Based on the code above, if the `momentum = 0.9`, the  `torch_momentum ` method is equivalent `momentum` ?

**Comment by lzhao4ever** [comment]
They are not the same. I am adding a fix to momentum.
